src/main/kotlin/org/wfanet/measurement/common/RenewedFlow.kt

```diff
@@ -53,5 +51,4 @@ fun <T> renewedFlow(
   reconnect: Duration,
   reconnectDelay: Duration,
   block: suspend () -> Flow<T>
-): Flow<T> =
-  renewedFlow(reconnect.toMillis(), reconnectDelay.toMillis(), block)
+): Flow<T> = renewedFlow(reconnect.toMillis(), reconnectDelay.toMillis(), block)
```

Comment:
I find this formatting a bit confusing when the parameters are split up, should we just convert this to a block body with `return renewedFlow(...)`?

Reply:
This is the case where I would be _less_ likely to convert to a block body, as the expression fits together with the last line of the signature.